zebedee-cms/src/main/java/com/github/onsdigital/zebedee/Init.java

```diff
@@ -19,7 +27,26 @@
 
     @Override
     public void init() {
-        DPLogger.init(new NopConfig());
+
+        try {
+            LogSerialiser serialiser = new JacksonLogSerialiser(true);
+            LogStore logStore = new MDCLogStore(serialiser);
+            Logger logger = new LoggerImpl("com.zebedee.app");
```

Comment:
This needs match the appender name in the logback.xml - `com.github.onsdigital.zebedee.cms`
https://github.com/ONSdigital/zebedee/blob/develop/zebedee-reader/src/main/resources/logback.xml.